utils/ci/corefile_in_gha.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cd daos || exit 1
+
+echo ::group::Check for corefiles
+if [ "$BASE_DISTRO" == "ubuntu" ]
+then
+    # "apport" corefile repo
+    COREFILE_DIR="/var/crash/"
+else
+    COREFILE_DIR="/var/lib/systemd/coredump/"
+fi
+if [ $(find '$COREFILE_DIR' -maxdepth 1 -type f | wc -l) == 0 ]
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Expressions don't expand in single quotes, use double quotes for that. [SC2016]